src/main/java/Bwead/TaskList.java

```diff
@@ -0,0 +1,50 @@
+package Bwead;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a location in a 2D space. A <code>Point</code> object corresponds to
+ * a coordinate represented by two integers e.g., <code>3,6</code>
+ */
+public class TaskList {
+
+    private static ArrayList<Task> currentList;
+
+    /*
```

Comment:
Why is it in comments? 🤔 
If not useful, can just delete it. 
It will make the code base look cleaner.